Type definition error when building portal frontend application.

### 💡 Description

When building our portal code, it is throwing a build error related to the type of the navitems property of our Header:

```
src/components/Header/Header.tsx:88:9 - error TS2322: Type 'NavItems[]' is not assignable to type '[{ id: string; label: string; href: string; items?: [{ id: string; label: string; href: string; }] | undefined; }]'.
  Target requires 1 element(s) but source may have fewer.

88         navItems={navItems}
           ~~~~~~~~

  ../../../wds-react/dist/esm/components/Header/Header.d.ts:2:5
    2     navItems: [
          ~~~~~~~~
    The expected type comes from property 'navItems' which is declared here on type 'IntrinsicAttributes & HeaderProps'
```

### ⚔️ Parent Epic / Related Tickets

_No response_

--- a/src/components/Header/Header.tsx
+++ b/src/components/Header/Header.tsx
@@ -2,7 +2,7 @@ import TitleBar from "../TitleBar/TitleBar";
 import Navbar from "../Navbar/Navbar";
 import Box from "@mui/material/Box";
 
-type NavItems = {
+export type NavItems = {
   id: string;
   label: string;
   href: string;

--- a/src/components/Header/Header.tsx
+++ b/src/components/Header/Header.tsx
@@ -2,21 +2,21 @@ import TitleBar from "../TitleBar/TitleBar";
 import Navbar from "../Navbar/Navbar";
 import Box from "@mui/material/Box";
 
-export type HeaderProps = {
-  navItems: [
+type NavItems = {
+  id: string;
+  label: string;
+  href: string;
+  items?: Array<
     {
       id: string;
       label: string;
       href: string;
-      items?: [
-        {
-          id: string;
-          label: string;
-          href: string;
-        },
-      ];
-    },
-  ];
+    }
+  >
+};
+
+export type HeaderProps = {
+  navItems: NavItems[];
   title: string;
   titleLink: string;
   subTitle?: string;

